packages/notifications/src/styled/StyledWell.spec.tsx

```diff
@@ -6,31 +6,71 @@
  */
 
 import React from 'react';
+import { css } from 'styled-components';
+import rgba from 'polished/lib/color/rgba';
+import stripUnit from 'polished/lib/helpers/stripUnit';
 import { render, renderRtl } from 'garden-test-utils';
-import { StyledWell } from './StyledWell';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { StyledWell, StyledTitle } from '../styled';
 
 describe('StyledWell', () => {
   it('renders default well styling', () => {
     const { container } = render(<StyledWell />);
+    const { borders, borderRadii, colors, space, fontSizes, palette } = DEFAULT_THEME;
 
-    expect(container.firstChild).toHaveClass('c-callout');
+    expect(container.firstChild).toHaveStyleRule('position', 'relative');
+    expect(container.firstChild).toHaveStyleRule('border', `${borders.sm} ${palette.grey[300]}`);
+    expect(container.firstChild).toHaveStyleRule('border-radius', borderRadii.md);
+    expect(container.firstChild).toHaveStyleRule('background-color', `${colors.background}`);
+    expect(container.firstChild).toHaveStyleRule('padding', `${space.md} ${space.xl}`);
+    expect(container.firstChild).toHaveStyleRule(
+      'line-height',
+      ((space.base * 5) / stripUnit(fontSizes.md)).toFixed(5)
+    );
+    expect(container.firstChild).toHaveStyleRule('color', palette.grey[600]);
+    expect(container.firstChild).toHaveStyleRule('font-size', fontSizes.md);
+    expect(container.firstChild).not.toHaveStyleRule('box-shadow');
```

Comment:
Large blocks of style assertions are similar to snapshot testing. They are brittle because they change when the designs change. I'll go ahead and remove this, and only assert against syles that are guarded by logic.